packages/compass-aggregations/src/modules/side-panel.ts

```diff
@@ -0,0 +1,34 @@
+import type { AnyAction } from 'redux';
+import { isAction } from '../utils/is-action';
+
+enum ActionTypes {
+  SidePanelToggled = 'compass-aggregations/sidePanelToggled',
+}
+
+type SidePanelToggledAction = {
+  type: ActionTypes.SidePanelToggled;
+};
+
+type State = {
+  isPanelOpen: boolean;
+};
+
+export const INITIAL_STATE: State = {
+  isPanelOpen: false,
+};
+
+export default function reducer(
```

Comment:
Should we also add tests for this? Although its very small piece of code but I guess it will bring down the coverage but upto you because I couldn't find coverage, do we report coverage for compass?